app/assets/v2/js/grants/fund.js

```diff
@@ -0,0 +1,203 @@
+/* eslint-disable no-console */
+$(document).ready(function() {
+
+  $('.js-select2').each(function() {
+    $(this).select2();
+  });
+
+  $('.select2-selection__rendered').hover(function() {
+    $(this).removeAttr('title');
+  });
+
+  updateSummary();
+
+  $('#js-fundGrant').validate({
+    submitHandler: function(form) {
+      var data = {};
+
+      $.each($(form).serializeArray(), function() {
+        data[this.name] = this.value;
+      });
+
+      $('#token_symbol').val($('#js-token option:selected').text());
+
+      let realPeriodSeconds = 0;
+
+      if (data.frequency) {
+
+        // translate timeAmount&timeType to requiredPeriodSeconds
+        let periodSeconds = data.frequency;
+
+        if (data.frequency_unit == 'days') {
+          periodSeconds *= 86400;
+        } else if (data.frequency_unit == 'months') {
+          periodSeconds *= 2592000;
+        }
+        if (periodSeconds) {
+          realPeriodSeconds = periodSeconds;
+        }
+      }
+
+      let deployedSubscription = new web3.eth.Contract(compiledSubscription.abi, data.contract_address);
+      let deployedToken;
+      let selected_token;
+
+      if (data.token_address != '0x0000000000000000000000000000000000000000') {
```

Comment:
Correct. However, `'0x0000000000000000000000000000000000000000'` is the 0 value for a contract and in `grants` is used to represent `Any Token` since user selects no contract when chosing this option in `grants/new`.